code/game/machinery/doors/airlock.dm

```diff
@@ -45,6 +45,9 @@
 	var/obj/item/weapon/airlock_electronics/electronics = null
 	var/hasShocked = 0 //Prevents multiple shocks from happening
 	var/autoclose = 1
+	var/sabotaged = 0 //Used by Syndicate door charges for boom boom
+	var/turf/sabotageTurf = null //The turf that will go boom

```

Comment:
Do we really need a var just for that? Can't we use T=get_turf(theairlock) for the explosion and use range(1, T) for the shrapnel?